library/translation.inc.php

```diff
@@ -263,6 +263,15 @@ function html_header_show() {
     //  We commented this out since we are now standardizing encoding
     //  in the globals.php file.
```

Comment:
A critical question I am trying to sort out here is the following:
1. Will this feature be stepping a stone for the more formal assets manager that you plan to bring in? Meaning will this function and global path still be used?